src/sidebar/components/UserMenu.js

```diff
@@ -16,16 +16,18 @@ import MenuSection from './MenuSection';
  * /
 
 /**
- * @typedef AuthState
- * @prop {'logged-in'|'logged-out'|'unknown'} status
+ * @typedef AuthStateLoggedIn
+ * @prop {'logged-in'} status
  * @prop {string} displayName
  * @prop {string} userid
  * @prop {string} username
+ * @prop {string} provider
```

Comment:
From what I could tell using grep and TS's "Find References" tool, nothing reads the provider field. Therefore we could just remove it from the type entirely.

Reply:
I thought about removing this unused property, but it leads to changing some tests. I will do that.